src/sideof.jl

```diff
@@ -67,6 +67,22 @@ Possible results are `IN` or `OUT` the `mesh`.
 """
 sideof(point::Point{3}, mesh::Mesh{3}) = sideof((point,), mesh) |> first
 
+# -----
+# K-SIMPLEX
+# -----
```

Comment:
The Simplex method should go in the GEOMETRY section above.

Reply:
Done.